lib/assembly-image/jp2_creator.rb

```diff
@@ -69,7 +69,7 @@ def overwrite?
       def jp2_create_command(source_path:, output:)
         options = []
         # TODO: Consider using ruby-vips to determine the colorspace instead of relying on exif (which is done below)
-        options << '-jp2_space sRGB' if image.samples_per_pixel == 3
+        options << '-jp2_space sRGB' if image.srgb?
```

Comment:
does this mean we don't even need the `samples_per_pixel` method?

Reply:
Yes. I rebased and removed that method.